app/src/main/java/org/mozilla/focus/settings/privacy/PreferenceToolTipCompose.kt

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.settings.privacy
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.preference.Preference
+import androidx.preference.PreferenceViewHolder
+import org.mozilla.focus.databinding.FocusPreferenceComposeLayoutBinding
+import org.mozilla.focus.ext.settings
+
+class PreferenceToolTipCompose(context: Context, attrs: AttributeSet?) :
+    Preference(context, attrs) {
+
+    override fun onBindViewHolder(holder: PreferenceViewHolder) {
+        super.onBindViewHolder(holder)
+        val binding = FocusPreferenceComposeLayoutBinding.bind(holder.itemView)
+        binding.toolTipContent.setContent {
+            ToolTipContent {
+                context.settings.shouldShowPrivacySecuritySettingsToolTip = false
```

Comment:
nit: For readability, I think we should keep the lambda in parentheses and maybe also use a named param to avoid any confusion caused by also having a composable here.

Reply:
Done